Dockerfile

```diff
@@ -36,34 +36,33 @@ COPY ./pkg/plugins/diff/delta_diff_server/Cargo.lock ./Cargo.lock
 COPY ./pkg/plugins/diff/delta_diff_server/Cargo.toml ./Cargo.toml
 
 # 3. Build only the dependencies to cache them in this layer
-RUN cargo build --release
+RUN RUSTFLAGS=-Ctarget-feature=-crt-static cargo build --release
```

Comment:
This flag is worth documenting (based on F2F explanations), perhaps by adding a link that explains something about why we need it.